app/models/course.rb

```diff
@@ -5,4 +5,13 @@ class Course < ActiveRecord::Base
   belongs_to :creator, class_name: User.name
   has_many :course_users, inverse_of: :course, dependent: :destroy
   has_many :users, through: :course_users
+
+  before_validation :set_default_values, on: :create
+
+  private
+
+  def set_default_values
```

Comment:
So we should document all the new methods which we added?

On Tue, Jan 6, 2015 at 9:39 AM, Joel Low <notifications@github.com> wrote:

> In app/models/course.rb
> <https://github.com/Coursemology/coursemology2/pull/25#discussion-diff-22503693>
> :
>
> > @@ -5,4 +5,13 @@ class Course < ActiveRecord::Base
> >    belongs_to :creator, class_name: User.name
> >    has_many :course_users, inverse_of: :course, dependent: :destroy
> >    has_many :users, through: :course_users
> > +
> > +  before_validation :set_default_values, on: :create
> > +
> > +  private
> > +
> > +  def set_default_values
>
> and document the method please...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Coursemology/coursemology2/pull/25/files#r22503693>.
>